src/costmap_to_lines_convex_hull.cpp

```diff
@@ -61,18 +61,19 @@ CostmapToLinesDBSMCCH::~CostmapToLinesDBSMCCH()
 void CostmapToLinesDBSMCCH::initialize(ros::NodeHandle nh)
 { 
     // DB SCAN
-    max_distance_ = 0.4; 
-    nh.param("cluster_max_distance", max_distance_, max_distance_);
+    parameter_.max_distance_ = 0.4; 
+    nh.param("cluster_max_distance", parameter_.max_distance_, parameter_.max_distance_);
     
-    min_pts_ = 2;
-    nh.param("cluster_min_pts", min_pts_, min_pts_);
+    parameter_.min_pts_ = 2;
+    nh.param("cluster_min_pts", parameter_.min_pts_, parameter_.min_pts_);
     
-    max_pts_ = 30;
-    nh.param("cluster_max_pts", max_pts_, max_pts_);
+    parameter_.max_pts_ = 30;
+    nh.param("cluster_max_pts", parameter_.max_pts_, parameter_.max_pts_);
     
     // convex hull
-    min_keypoint_separation_ = 0.1;
-    nh.param("convex_hull_min_pt_separation", min_keypoint_separation_, min_keypoint_separation_);
+    parameter_.min_keypoint_separation_ = 0.1;
```

Comment:
Can be done on one line puttin 0.1 as default param

Reply:
followed the pattern of the existing code :)